### ISSUE TITLE
Chocolatey packages

### ISSUE THREAD

**Issue by Dexusan** [Syncplay/syncplay#358]
Any plans to release on chocolatey? I use it for a majority of my programs and having syncplay on it would be nice.
https://chocolatey.org/

**Comment by Et0h** [comment]
We have no plans to release on Chocolatey. Syncplay only has a small base of regular developers (all unpaid volunteers) and so we're not currently looking to increase our development, maintenance and support burdens by supporting more repository and distribution options. Most of the packaging systems we currently support are because we ourselves use them, and when we've ventured beyond that it has often been more of a burden than we had anticipated.

**Comment by digitalcoyote** [comment]
I'd be willing to create and maintain a chocolatey pacakge for this (No changes on your end necessary). I have an [AU repo](https://github.com/digitalcoyote/chocolatey-packages) that would watch your repo for releases and build/deploy packages when new releases were detected.

**Comment by ruffsl** [comment]
This would be awesome! When setting up a group of friends or family who use windows for a movie night remotely, it's a lot easier to walk them through pasting one command to a package manager, then guiding them through `N` installation wizards.

E.g. pasting this would just about do it all from one admin console:

```
Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
choco install mpv youtube-dl discord syncplay
```

**Comment by Et0h** [comment]
We don't have the capacity to create our own Chocolatey package, but anyone can make their own unofficial package because Chcolatey packages don't need to be created by the original developers.

However, for someone to run their own Chocolatey package the first step they may need to take is to contact the author of https://chocolatey.org/packages/syncplay or the Chocolatey team to ask them to remove the broken attempt to add Syncplay to Chocolatey. The maintainer for that is [kdominik](https://chocolatey.org/profiles/kdominik). It is possible that you will be able to collaborate with them.

**Comment by digitalcoyote** [comment]
I'll try to start that process soon (kinda busy at the moment). In my experience this takes a while, so I wouldn't expect an update on Chocolatey til after the new year unless the maintainer is responsive.

**Comment by digitalcoyote** [comment]
No response yet. I'll give it some more time as Chocolatey Admins are volunteers and usually very busy with work/holidays this time of year. Eventually if the maintainer doesn't respond it can be taken through the admins.

**Comment by albertosottile** [comment]
@digitalcoyote Any news on this? 

In any case, I am closing this issue as for the moment we do not intend to officially support and maintain this release channel. Anyone is free to provide any other release channel (e.g. the AUR package for Arch) but we will not publish its artifacts/links on GitHub or on the website.

**Comment by digitalcoyote** [comment]
@albertosottile https://chocolatey.org/packages/syncplay/1.6.7 

Installable immediately if you specify the version like
 `choco install syncplay --version=1.6.7`

Takeover was approved today but moderation can take weeks if the volunteer moderators are busy. I did notice that this appears to use NSIS, but was ignoring the /SD silent install flag. That might make it a bit more difficult for some users. It's not a big problem as they allow non-silent installs, but the preferred method is allowing a user to install/update silently.

I'll look into this eventually if no one else does (my python is kinda rusty and it may be a few months before I have time to circle back to this). I'll leave it up to you if that warrants opening an issue or not.

**Comment by atagulalan** [comment]
Hey @digitalcoyote could you check out https://github.com/Syncplay/syncplay/pull/460? I hope this fixes silent install problem.

Example usage: `.\Syncplay-X.X.X-Setup.exe /S /LANG=1033`

**Comment by digitalcoyote** [comment]
I'll look into when I get a chance, if it's not merged yet, I may

**Comment by digitalcoyote** [comment]
Working on getting a windows environment setup to build so I can test it with chocolatey. Might have some results tonight since I won't have time to finish this morning

**Comment by digitalcoyote** [comment]
@atagulalan Worked! I'll see how the Chocolatey Moderators feel about a patched installer to get 1.6.9 out. They may prefer I wait for the PR to be merged.
